x/programs/cmd/simulator/cmd/config.go

```diff
@@ -8,17 +8,12 @@ import (
 	"fmt"
 )
 
-const (
-	ProgramCreate  = "program_create"
-	ProgramExecute = "execute"
-)
```

Comment:
Were these not being used? Don't we have linter warnings for this type of stuff?

Reply:
hmm, I believe they might have been used just not anymore

Reply:
the `ProgramCreate` was used since we used to check if the request was a program creation from the method name, which was really odd.